src/utils/address.cljs

```diff
@@ -36,6 +36,11 @@
   (when value
     (str (subs value 0 6) "\u2026" (subs value (- (count value) 3) (count value)))))
 
+(defn get-short-wallet-address
+  [value]
+  (when value
+    (str (subs value 0 5) "\u2026" (subs value (- (count value) 3) (count value)))))
```

Comment:
any idea what `"\u2026" is? perhaps it's better we put it in a named def

Reply:
"\u2026" is three dots `...`. I am following the convention in that file.

Reply:
Yeah that would be great, we can update the other uses in the file too 👍